src/components/App/App.js

```diff
@@ -1,38 +1,41 @@
 import React, { Component } from 'react';
 import queries from '../../utils/queries/queries';
+import { LoginForm } from '../LoginForm/LoginForm';
 
-export class App extends Component {
+class App extends Component {
```

Comment:
I know Nim had added the export in anticipation that App would get connected to the store but to clear the linting error I reverted it back to using just the default export. I also am not quite sure that we will end up connecting App to Redux.

Reply:
Agreed, once everything is set up I don't see a reason for app to be connected